xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+     and <literal>443</literal>.
+    </para>
+    <note>
+     <title>Additional Port Configuration</title>
+     <para>
+      If you choose to add more container image registries to your internal
+      network, these must run on different ports than the standard registry
+      running on <literal>5000</literal>. Configure your network to allow for
+      this communication accordingly.
+     </para>
+    </note>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.storage">
+    <title>Trusted Storage</title>
+    <para>
+     Transferring data from the external network mirror to the internal mirror
+     can be performed in many ways. The most common way is portable storage (USB
+     keys or external hard drives).
+    </para>
+    <para>
+     Sizing of the storage is dependent on the number of data sources that need
+     to be stored. Container images can easily measure several Gigabytes per
+     item; although they are generally smaller for &kube; related
+     applications. The overall size of any given RPM repository is at least tens
+     of Gigabytes. For example: At the time of writing, the package repository
+     for &sls; contains approximately <literal>36 GB</literal> of data.
+    </para>
+    <para>
+     The storage must be formatted to a filesystem type supporting files larger
+     than <literal>4 GB</literal>.
+    </para>
+    <para>
+     The type of filesystem used for the storage depends on your requirements.
+     We recommend <literal>btrfs</literal>.
+    </para>
+    <para>
+     We recommend external storage with at least <literal>128 GB</literal>.
+    </para>
+    <note>
+     <title>Mount Point For Storage In Examples</title>
+     <para>
+      In the following procedures, we will assume the storage (when connected)
+      is mounted on <filename>/mnt/storage</filename>. Please make sure to
+      adjust the mountpoint in the respective command to where the device is
+      actually available.
+     </para>
+    </note>
+    <note>
+     <title>Handling Of Trusted Storage</title>
+     <para>
+      Data integrity checks, duplication, backup, and secure handling procedures
+      of trusted storage are beyond the scope of this document.
+     </para>
+    </note>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.rpm-repository">
+   <title>RPM Repository Mirror</title>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.rpm-repository.mirror">
+    <title>Mirror Configuration</title>
+    <note>
+     <title>Deploy The Mirror Before &productname; Cluster Deployment</title>
+     <para>
+      The mirror on the airgapped network must be running and populated before
+      deploying &productname;. Starting with &productname; v4 many components
+      will no longer be shipped with the install media and must be fetched
+      from the repository. For v3 this significantly simplifies deployment since
```

Comment:
Should this say 'v4' ?

Reply:
Is v4 released? When will it be released? Is this feature only contained in v4? So far I wasn't told that this is separate. If we need to release this as v4 only there is lots of other work that needs to go into docs before we can do this.

Reply:
This paragraph already mentions `v4` a few lines above and I am not sure why it's there. But the simplification that's mentioned in the last sentence probably affects >= `v4` instead of `v3` that's why I mentioned that here. @flavio ?

Reply:
Yes, this is going to affect releases of CaaSP after v3. So v4 will be the first one; it hasn't been released yet.

Reply:
So we need to replace `For v3 this...` with `For v4 this...` or drop the entire comment about the upcoming v4 release.

Reply:
Removed the comment